docs/src/components/productX/XDateRangeDemo.tsx

```diff
@@ -166,7 +166,7 @@ export default function XDateRangeDemo() {
           }}
         >
           <Typography variant="body2" fontWeight="bold" sx={{ mr: 1 }}>
-            🎉&nbsp;&nbsp;&nbsp;Stable version available now for your project!
+            🎉&nbsp;&nbsp;&nbsp;Date Range Picker now available for your project!
```

Comment:
It leaves the same feeling on my end.

Maybe we should have some controls to change the content of the demo like showing a date field, a date input, etc.

Reply:
Sounds good! @danilo-leal do you have any design preference for that?